client/components/main/header/Header.jsx

```diff
@@ -31,10 +38,23 @@ const Header = () => {
             <span className="beta-tag">BETA</span>
           </div>
         </Link>
+        <a // eslint-disable-line
+          role="button"
+          tabIndex={0}
+          className={clx('navbar-burger', { 'is-active': activeBurger })}
+          aria-label="menu"
+          aria-expanded="false"
+          onClick={handleClick}
+          onKeyDown={handleClick}
+        >
+          <span aria-hidden="true" />
+          <span aria-hidden="true" />
+          <span aria-hidden="true" />
+        </a>
       </div>
 
-      <div id="navbar" className="navbar-menu">
-        <div className="navbar-end">
+      <div id="navbar" className={clx('navbar-menu', { 'is-active': activeBurger })}>
+        <div className="navbar-start">
```

Comment:
Was this done to split the core tool links and the others (about/contact) onto the left/right sides of the menu bar? If so, a few thoughts:

I get the idea behind it but I think it presents some UX problems. The user needs to move their eyes across the entire width of the screen to view all links. Seems like this could create some confusion and increase the likelihood of a casual or less web-savvy user missing some parts of our site. I prefer consolidating all links on one side but I'm curious to hear your thoughts on this.

Let's get some thoughts from @danarchen @kevindperez

Reply:
Screenshot for @danarchen @kevindperez :

<img width="1677" alt="Screen Shot 2020-09-09 at 5 07 39 PM" src="https://user-images.githubusercontent.com/40484278/92668290-421d9180-f2c3-11ea-8b74-87ae86180611.png">